tensorflow/core/common_runtime/local_device.h

```diff
@@ -47,6 +59,8 @@ class LocalDevice : public Device {
   struct EigenThreadPoolInfo;
   std::unique_ptr<EigenThreadPoolInfo> owned_tp_info_;
 
+  const GPUOptions::STREAM_MERGE_OPTIONS gpu_stream_merge_options_;
```

Comment:
Can we move this to BaseGPUDevice? LocalDevice is the base class for multiple different devices.